src/components/grapedown/render.js

```diff
@@ -69,7 +69,8 @@ class Renderer {
     return renderInlineImage(attrs.src, formatMessage(message, values))
   }
 
-  render() {
+  render(props) {
+    this.props = props
```

Comment:
Do we need this? I didn't see any cases of using this.
But, if needed it's ok. I just didn't check this pr on my env.